cmd/kubeadm/app/cmd/phases/reset/updateclusterstatus.go

```diff
@@ -44,7 +44,10 @@ func runUpdateClusterStatus(c workflow.RunData) error {
 	// Reset the ClusterStatus for a given control-plane node.
 	cfg := r.Cfg()
 	if isControlPlane() && cfg != nil {
-		uploadconfig.ResetClusterStatusForNode(cfg.NodeRegistration.Name, r.Client())
+		err := uploadconfig.ResetClusterStatusForNode(cfg.NodeRegistration.Name, r.Client())
```

Comment:
can be:
```
if err := uploadconfig.ResetClusterStatusForNode(cfg.NodeRegistration.Name, r.Client()); err != nil {
	return err
}
```

SGTM
cc @yagonobre

Reply:
@neolit123 thank you for the review. I've updated the PR accordingly.